backend/windmill-api/src/settings.rs

```diff
@@ -701,40 +701,89 @@ async fn setup_ducklake_catalog_db_inner(
         sslmode = ssl_mode
     );
 
-    let (client, connection) = tokio::time::timeout(
```

Comment:
@diegoimbert I think we can take over the PR, it's mostly to make it easier for us from @knowsuchagency